tests/btc/transactions.test.ts

```diff
@@ -277,9 +277,29 @@ describe('bitcoin transactions', () => {
     const ordinalAddress = 'bc1prtztqsgks2l6yuuhgsp36lw5n6dzpkj287lesqnfgktzqajendzq3p9urw';
     const btcAddress = '1H8voHF7NNoyz76h9s6dZSeoypJQamX4xT';
 
-    const { fee } = await getBtcFeesForOrdinalSend(recipientAddress, ordinalOutputs[0], btcAddress, network, [
+    const ordinalUtxos = [
```

Comment:
This won't really test the filter though because this UTXO is not in the mock list of all UTXOs for the address.

Could you try make these changes so that we actually test the filtering 🙏 :
- add this new UTXO to ordinalOutputs ion line 239 above
- on line 253, the utxo mock data, change it to:
```
const utxos: Array<UTXO> = [
      {
        txid: '1f2bbb92a74d379db2502e8ae7a57917041db5dc531ef54e64ca532aa9f59d8c',
        vout: 2,
        status: {
          confirmed: true,
          block_height: 123123,
          block_time: 1677048365,
          block_hash: '000000000000000000072266ee093771d806cc9cb384461841f9edd40b52b67f',
        },
        value: unspent1Value,
      },
     ...ordinalOutputs,
    ];
```

The resulting fee should remain the same because those 2 UTXOs should get filtered out.